### PR TITLE
Usage records, PDF, Custom api calls

### LINKED ISSUE #2: Retrieve a PDF Invoice
Looks like there are no method to retrieving PDF for Invoice (https://dev.recurly.com/docs/retrieve-a-pdf-invoice).
Is it possible to get PDF now?

### PR DESCRIPTION
**Pull request by mrbar42**
fixes #2

### CODE REVIEW

**Comment by umayr** [comment]
Thanks for the PR. It looks good. Give me a few hours to look deep into it.

File: promise.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var Promise = require('bluebird');
```
Comment (umayr) [inline promise.js]: As `Promises` are now available on native node, it'd make more sense if  `bluebird` is conditionally required?

**Comment by mrbar42** [comment]
Could by any chance deploy the latest version to npm?

**Comment by umayr** [comment]
I'll do it right away. I was away for the weekend.

**Comment by umayr** [comment]
Published as `2.3.0`

**Comment by mrbar42** [comment]
Cheers :+1:

**Pull request merged**